docs/page-layouts.md

```diff
@@ -0,0 +1,45 @@
+# Page Layouts
+
+Page layouts allow people to apply individual pages defined in templates to their story without choosing an entire template. Page layouts are based on the same JSON definitions as templates defined in `src/dashboard/templates/raw`.
+
+## How Page Layouts differ from Template Pages
+
+Since page layouts are based on the same source files as page layouts, some customization of raw templates is required for them to work correctly with Page Layouts.
```

Comment:
```suggestion
Since page layouts are based on the same source files as templates, some customization of raw templates is required for them to work correctly with Page Layouts.
```